thirdparty/clamav/zimbra-clamav/debian/control

```diff
@@ -1,6 +1,6 @@
 Source: zimbra-clamav
 Build-Depends: debhelper (>= 9), m4, dpkg-dev (>= 1.15.7), zimbra-openssl-dev (>= 1.1.1h-1zimbra8.7b4ZAPPEND), zimbra-libxml2-dev,
- libz-dev, zimbra-libmilter-dev, libncurses5-dev
+ libz-dev, zimbra-libmilter-dev, libncurses5-dev, libpcre2-dev, libjson-c-dev, libbz2-dev, check
```

Comment:
don't we need to install rust toolchain for compilation?

Reply:
yes, we need to install rust toolchain. But it is not provided by package. There is a script to install rust toolchain.

Reply:
https://www.rust-lang.org/tools/install

Reply:
can we update makefile to install rust first if not present, I would like to reduce package dependecies to be installed in system when compiling these packages, so we can port this easily to circleci